tools/Cron_jobs/quarantine_maint.php

```diff
@@ -84,6 +84,11 @@ function quarantine_clean()
                         // Update the quarantine flag
                         $sql = "UPDATE maillog SET timestamp=timestamp, quarantined = NULL WHERE id='$id'";
                         dbquery($sql);
+                        //If auto quarantine release is enabled, remove from autorelease table when quarantined email expires
+                        if (AUTO_RELEASE) {
```

Comment:
When introducing new constants in config file it's better to check for existence before:
```php
if (defined('AUTO_RELEASE') && AUTO_RELEASE === true) {
```